packages/kbn-search-connectors/components/configuration/connector_configuration.tsx

```diff
@@ -95,7 +103,10 @@ export const ConnectorConfigurationComponent: React.FC<ConnectorConfigurationPro
   const [isEditing, setIsEditing] = useState(false);
 
   useEffect(() => {
-    setIsEditing(false);
+    if (!isDeepEqual(configuration, configurationRef.current)) {
```

Comment:
I have no context for this so these might be dumb questions:

Is it possible we could leave `isEditing === true` with this conditional?

Whats the benefit of have this separate `useEffect()` to set `isEditing` to false vs an `else` condition in the other `useEffect` that sets `isEditing` to true?

Reply:
Fair point, I consolidated the two.